lib/interface/cli/Command.js

```diff
@@ -296,7 +296,7 @@ class Command {
         yargsCommand.description = this.description;
         yargsCommand.command = this.command;
         yargsCommand.builder = this._createBuilderFunction();
-        yargsCommand.handler = wrapHandler(this.handler, this.requiresAuthentication);
+        yargsCommand.handler = wrapHandler(this.handler || (() => yargs.showHelp()), this.requiresAuthentication);
```

Comment:
When we don't have handler on command and do something like
codefresh create asdf
it throw an error despite showing help